DSharpPlus/Entities/Guild/DiscordRole.cs

```diff
@@ -57,6 +57,25 @@ public DiscordColor Color
         [JsonProperty("hoist", NullValueHandling = NullValueHandling.Ignore)]
         public bool IsHoisted { get; internal set; }
 
+        /// <summary>
+        /// The url for this role's icon, if set.
+        /// </summary>
+        public string IconUrl => this.IconHash != null ? $"https://cdn.discordapp.com/role-icons/{this.Id}/{this.IconHash}.png" : null;
+
+        /// <summary>
+        /// The hash of this role's icon, if any.
+        /// </summary>
+        [JsonProperty("icon")]
+        public string IconHash { get; internal set; }
+
+        /// <summary>
+        /// The emoji associated with this role's icon, if set.
+        /// </summary>
+        public DiscordEmoji Emoji => this._emoji != null ? DiscordEmoji.FromUnicode(this._emoji) : null;
+
+        [JsonProperty("emoji")]
```

Comment:
```suggestion
        [JsonProperty("unicode_emoji")]
```

It is unicode_emoji https://github.com/discord/discord-api-docs/blob/master/docs/resources/Guild.md#json-params-9

Reply:
Still this